rpc/api.py

```diff
@@ -274,9 +276,51 @@ def import_submission(request, document_id, rpcapi: rpcapi_client.DefaultApi):
         )
 
     # Create the RfcToBe
-    serializer = CreateRfcToBeSerializer(data=request.data, context={"draft": draft})
+    serializer = CreateRfcToBeSerializer(data={**request.data, "draft": draft.pk})
```

Comment:
An alternative syntax for dict merging is
```python
data=request.data | {"draft": draft.pk}
```
(I find it somewhat more readable, though not everyone is familiar with it)